MarvellOS.py

```diff
@@ -603,6 +604,24 @@ def sample(self):
         self.samwin.bind("<Left>", self.lkey)
         self.samwin.bind("<Right>", self.rkey)
 
+        self.total=len(list(glob.glob('./sample_pictures/*.gif')))
+        self.chooseFile = Button(self.samwin, text ="Upload Image",command =  lambda: self.upload_image(), padx = 10,pady = 10, font=("Bauhaus 15 ",10))
+        self.chooseFile.place(x=100,y=310)
+
+        
+
+    def upload_image(self):
+        
+        filename = tkfd.askopenfilename(filetypes =[('Image Files', '*.gif')])
+        if filename != () and filename != "":
+            self.img =Image.open(filename).resize(self.new_size,Image.ANTIALIAS)
+            self.total+=1
+            self.img.save('./sample_pictures/sample{}.gif'.format(self.total))
+            self.img=ImageTk.PhotoImage(self.img)
+            self.pics["image"]=self.img
+
+
+
     def lkey(self, event):
         if self.i != 1:
```

Comment:
You'll need to make the `rkey` and `lkey` dynamic too, since currently they only check for 10 images.